packages/react-next/src/utilities/grid/Grid.base.tsx

```diff
@@ -1,96 +1,77 @@
+    styles,
+    doNotContainWithinFocusZone,
+  } = props;
 
-  public render(): JSX.Element {
-    const props = this.props;
-    const {
-      items,
-      columnCount,
-      onRenderItem,
-      // eslint-disable-next-line deprecation/deprecation
-      ariaPosInSet = props.positionInSet,
-      // eslint-disable-next-line deprecation/deprecation
-      ariaSetSize = props.setSize,
-      styles,
-      doNotContainWithinFocusZone,
-    } = props;
+  const htmlProps = getNativeProps<React.HTMLAttributes<HTMLTableElement>>(
+    props,
+    htmlElementProperties,
+    // avoid applying onBlur on the table if it's being used in the FocusZone
+    doNotContainWithinFocusZone ? [] : ['onBlur'],
+  );
 
-    const htmlProps = getNativeProps<React.HTMLAttributes<HTMLTableElement>>(
-      this.props,
-      htmlElementProperties,
-      // avoid applying onBlur on the table if it's being used in the FocusZone
-      doNotContainWithinFocusZone ? [] : ['onBlur'],
-    );
+  const classNames = getClassNames(styles!, { theme: props.theme! });
 
-    const classNames = getClassNames(styles!, { theme: this.props.theme! });
+  // Array to store the cells in the correct row index
 
-    // Array to store the cells in the correct row index
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const rowsOfItems: any[][] = toMatrix(items, columnCount);
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const rowsOfItems: any[][] = toMatrix(items, columnCount);
 
-    const content = (
-      <table
-        aria-posinset={ariaPosInSet}
-        aria-setsize={ariaSetSize}
-        id={this._id}
-        role="grid"
-        {...htmlProps}
-        className={classNames.root}
-      >
-        <tbody>
-          {rowsOfItems.map((rows: [], rowIndex: number) => {
-            return (
-              <tr role={'row'} key={this._id + '-' + rowIndex + '-row'}>
-                {rows.map((cell, cellIndex: number) => {
-                  return (
-                    <td
-                      role={'presentation'}
-                      key={this._id + '-' + cellIndex + '-cell'}
-                      className={classNames.tableCell}
-                    >
-                      {onRenderItem(cell, cellIndex)}
-                    </td>
-                  );
-                })}
-              </tr>
-            );
-          })}
-        </tbody>
-      </table>
-    );
+  const content = (
+    <table
+      aria-posinset={ariaPosInSet}
+      aria-setsize={ariaSetSize}
+      id={id}
+      role="grid"
+      {...htmlProps}
+      className={classNames.root}
+    >
+      <tbody>
+        {rowsOfItems.map((rows: [], rowIndex: number) => {
+          return (
+            <tr role={'row'} key={rowIndex + '-row'}>
+              {rows.map((cell, cellIndex: number) => {
+                return (
+                  <td role={'presentation'} key={cellIndex + '-cell'} className={classNames.tableCell}>
```

Comment:
While you're here this can be a little simpler:
```suggestion
        {rowsOfItems.map((rows, rowIndex) => {
          return (
            <tr key={rowIndex}>
              {rows.map((cell, cellIndex) => {
                return (
                  <td role="presentation" key={cellIndex} className={classNames.tableCell}>
```

Reply:
I'm not sure if role={'row'} should be removed since it is used to get the table rows in the tests.